src/picongpu/include/particles/manipulators/RatioWeightingImpl.hpp

```diff
@@ -73,9 +73,9 @@ struct RatioWeightingImpl
         if (isDesParticle && isSrcParticle)
         {
             const float_X densityRatioDes =
-                traits::GetDensityRatio<T_DesParticle>::type::getDefaultValue();
+                traits::GetDensityRatio<T_DesParticle>::type::getValue();
```

Comment:
ah no, your change is invalid. we use `value_identifier` in `speciesDefinition` which is a "default" value for an identifier that *could* be an l value, but is not used as such in the species.